pkg/export/otel/metrics.go

```diff
@@ -59,6 +59,7 @@ const (
 	FeatureSpan        = "application_span"
 	FeatureGraph       = "application_service_graph"
 	FeatureProcess     = "application_process"
+	FeatureEBPF        = "ebpf"
```

Comment:
Since this feature is not monitoring any actual app/host target but Beyla internally, instead of enabling eBPF metrics here, I would create a new `internal_metrics` section where you can select ebpf and other groups of internal metrics.

Reply:
> Since this feature is not monitoring any actual app/host target but Beyla internally,

Is monitoring any BPF programs loaded in the host. Is more than internal metrics.